NEWS.md

```diff
@@ -1,5 +1,10 @@
 # serocalculator (development version)
 
+# serocalculator 1.2.9.9007
+* Add `as_noise_params` (#228) 
+
+* Updated enteric fever example article with upgraded code and visualizations
```

Comment:
I don't see any changes to the enteric fever example article in this PR, and I don't think that we need them to be in this PR; did this line get accidentally added from a different PR?

Reply:
The line got added to this PR when the branch was updated.